src/components/linkMenu.js

```diff
@@ -29,15 +29,35 @@ import {
 } from '@ant-design/icons';
 import SubMenu from "antd/es/menu/SubMenu";
 import { withRouter } from "react-router";
+import Toggle from '../theme/Toggle';
+import lightVars from "../theme/light.json";
+import darkVars from "../theme/dark.json";
 
 
 class LinkMenu extends React.Component {
 
     constructor(props) {
         super(props);
         this.state = {
-            ProgramTracks: []
+            ProgramTracks: [],
+            theme: window.localStorage.getItem("theme") === "0" ? "light" : window.localStorage.getItem("theme")
         }
+        this.switchTheme();
+    }
+
+    themeToggle() {
+        this.setState({
+            theme: this.state.theme === 'light' ? 'dark' : 'light'
+        }, this.switchTheme);
+    }
+
+    switchTheme() {
+        window.localStorage.setItem("theme", this.state.theme);
+        window.less.modifyVars(this.state.theme === "light" ? lightVars : darkVars).then(() => {
```

Comment:
This appears to default in the wrong direction. Perhaps the following condition would be better:

`this.state.theme === "dark" ? darkVars : lightVars`